src/github-issues-prs.ts

```diff
@@ -427,4 +432,16 @@ export class GitHubIssuesPrsProvider implements TreeDataProvider<TreeItem> {
 		return Object.keys(remotes)
 			.map(key => remotes[key]);
 	}
+
+	private getAPIOption() {
+		if (this.host === "github.com") {
```

Comment:
Could you use single quotes `'` instead of double quotes? Just for consistency with the rest of the code.